diff --git a/node_modules/@types/react-mentions/index.d.ts b/node_modules/@types/react-mentions/index.d.ts
index c2aaa54..8cc261c 100644
--- a/node_modules/@types/react-mentions/index.d.ts
+++ b/node_modules/@types/react-mentions/index.d.ts
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-export {};
+export { };
 
 /**
  * MentionsInput is the main component rendering the textarea control. It takes one or multiple Mention components as its children.
@@ -120,7 +120,7 @@ export interface MentionItem {
 /**
  * The shape of suggestion items.
  */
-export interface SuggestionDataItem {
+export interface any {
     id: string | number;
     display?: string;
 }
@@ -145,5 +145,5 @@ export type OnChangeHandlerFunc = (
  */
 export type DataFunc = (
     query: string,
-    callback: (data: SuggestionDataItem[]) => void,
+    callback: (data: any[]) => void,
 ) => Promise<void> | void | Promise<SuggestionDataItem[]> | SuggestionDataItem[];
